### PR TITLE
Add mtu to UserInterface

### PR DESCRIPTION
**Pull request by teawater**

### CODE REVIEW

**Comment by Jimmy-Xu** [comment]
Can one of the admins verify this patch?

**Comment by gnawux** [comment]
ok to test, hykins

File: daemon/pod/networks.go [hunk 1]
```diff
@@ -62,6 +62,7 @@ func (inf *Interface) prepare() error {
 		Ip:      inf.spec.Ip,
 		Mac:     inf.spec.Mac,
 		Gw:      inf.spec.Gateway,
+		Mtu:	 inf.spec.Mtu,
```
Comment (gnawux) [inline daemon/pod/networks.go]: add `Mtu` in line 53 as well.

**Comment by gnawux** [comment]
retest this please, hykins

**Comment by Jimmy-Xu** [comment]
Can one of the admins verify this patch?

**Comment by gnawux** [comment]
lgtm

**Pull request merged**